inc/class-destination-s3.php

```diff
@@ -1110,11 +1110,11 @@ function awsgetbucket() {
 						s3bucketselected: $( 'input[name="s3bucketselected"]' ).val(),
 						s3base_url      : $( 'input[name="s3base_url"]' ).val(),
 						s3region        : $( '#s3region' ).val(),
-						s3base_region      : $( 'input[name="s3base_region"]' ).val(),
-						s3base_version      : $( 'input[name="s3base_version"]' ).val(),
-						s3base_signature      : $( 'input[name="s3base_signature"]' ).val(),
-						s3base_multipart      : $( 'input[name="s3base_multipart"]' ).is(':checked'),
-						s3base_pathstyle      : $( 'input[name="s3base_pathstyle"]' ).is(':checked'),
+						s3base_region          : $( 'input[name="s3base_region"]' ).val(),
```

Comment:
I'd actually rather not align the colons, but I know this is how it already was in the source code, so I'll fix this in the next version.